src/odin/maneuversbuilder.cc

```diff
@@ -1157,7 +1157,11 @@ void ManeuversBuilder::InitializeManeuver(Maneuver& maneuver, int node_index) {
 
   // Set the end level ref
   if (curr_edge && !curr_edge->GetLevelRef().empty()) {
-    maneuver.set_end_level_ref(curr_edge->GetLevelRef());
+    if (curr_edge->GetLevelRef().size() > 1) {
```

Comment:
If there is more than one `level_ref` value for the current edge, set `end_level_ref` to empty, so the instruction would be something like "Take the stairs" if the maneuver ends on stairs.